libs/src/services/creatorNode/CreatorNodeSelection.js

```diff
@@ -191,16 +194,16 @@ class CreatorNodeSelection extends ServiceSelection {
           this.currentVersion,
           resp.response.data.data.version
         )
-        isHealthy = isUp && versionIsUpToDate
+        const { storagePathSize, storagePathUsed } = resp.response.data.data
```

Comment:
can you either add a comment above this line or update the comment on L189

also this is super clean